authentication-guides/jira/README.md

```diff
@@ -0,0 +1,267 @@
+
+    ![Screenshot 2024-01-09 at 5.37.20 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.37.20_PM.png)
+
+3. Once it has been created, go back and click into the Event you just created.  You should now see an Event ID.  Copy that to your clipboard - ‼️ we will refer to this as the EventID going forward and will use it later in the exercise.
+
+    ![Screenshot 2024-01-09 at 5.39.03 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.39.03_PM.png)
+
+
+## Step 2: JIRA - Identify a Project with Approvals
+
+In this step we will do the required research in order to not only identify which project we are targeting to notify approvals for, but also to inspect the payload for issues within that project in order to build out the JIRA Automation effectively.
+
+### Identify the project and Request Types with Approvals
+
+1. Login to your JIRA ServiceDesk and choose a project that has request types with approvals.
+2. You can then choose “Project Settings” within the project to view the Request Types.
+3. Those with approvals should have a workflow under “Edit Workflow” that looks something like the screenshots below
+
+    ![Screenshot 2024-01-09 at 5.45.06 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.45.06_PM.png)
+
+
+![Screenshot 2024-01-09 at 5.46.41 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.46.41_PM.png)
+
+### Create a new issue of the identified request type
+
+The point of this step is to create a new issue type that will generate pending approvals.  This will allow us to view a fresh instance of that issue knowing that all the proper approvals can be read.
+
+1. Create an issue of the type you inspected in step 1.
+
+    ![Screenshot 2024-01-09 at 5.51.36 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.51.36_PM.png)
+
+    ![Screenshot 2024-01-09 at 5.52.08 PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.52.08_PM.png)
+
+2. You can now see I have an issue of identifier “LEG-54”.  It also has a pending approval attached to it.  Now I can leverage the JIRA API with Basic Auth in Postman to inspect the details of this issue using the API endpoint in the screenshot below which is also referenced in the [JIRA API Specification](https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-issueidorkey-get).  Copy and paste this JSON response into a JSON editor or simply leave it up in Postman so we can reference it later.
+
+![Screenshot_2024-01-09_at_5.56.10_PM.png](Use%20Case%20Guide%20JIRA%20Approvals%20f1ce3fe611e644a99a97f00cfb719fbb/Screenshot_2024-01-09_at_5.56.10_PM.png)
```

Comment:
Why is this step necessary? Just to have the custom field IDs so we can build an event notification in the future. It would be helpful to make that more clear

Reply:
@ajaymerchia It's so that the builder of this can identify additional fields from the JIRA incident they may want to map in the message.